### DISCUSSION TITLE
I made a swap with aperture swap, and didn't receive my tokens

### DISCUSSION THREAD

**Discussion by hakimm75019** [blockscout/blockscout#9814] [Support]
Hello , 
yesterday morning I have made a swap of 2070 usdc to stone eth, but i never received my tokens. 
I choose aperture swap directly but they don't understand the problem, 
1st : https://manta.layerbank.finance/bank
2nd : https://app.aperture.finance/swap?chainId=169
3rd : i made my swap

i directly choose apperture on the layerbank page on swap.

![capture blocksout](https://github.com/blockscout/blockscout/assets/156819806/0ee64d76-bed9-4276-b801-e9e56b204d97)
![capture ecran apertureswap](https://github.com/blockscout/blockscout/assets/156819806/8c130904-2e09-448b-80b5-a9f5069fc65c)

I hope that you will help me to find my money back, because it is clearly a bug

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
